docs/COMPILER_VERSIONS.md

```diff
@@ -63,6 +63,6 @@ We currently use the following version:
 
 | Type                     | Rust version | Note                              |
 | ------------------------ | ------------ | --------------------------------- |
-| Production Rust compiler | 1.73.0       | Builders version 0017             |
+| Production Rust compiler | 1.77.0       | Builders version 0019             |
 | Min Rust compiler        | 1.70.0       | Supports builder versions >= 0017 |
```

Comment:
Unrelated to this PR, but I think we need to bump the min version, since cosmwasm-vm requires `1.73` because of wasmer (and that's also what we use in CI)

Reply:
Yeah, right. This was done in CI already. Just a docs change. Will add here.